avocado/utils/build.py

```diff
@@ -82,7 +82,7 @@ def make(path, make='make', env=None, extra_args='', ignore_status=None,
 
     kwargs = dict(env=env,
                   ignore_status=ignore_status,
-                  allow_output_check=allow_output_check)
+                  allow_output_check=allow_output_check, encoding=encoding)
```

Comment:
I see your problem, but I don't really like adding yet another argument when there already is the `process_kwargs`. Also it only attempts to cover the fact that `process` uses incorrect encoding by default and it's drainers fail on binary data. Both is covered by: https://github.com/avocado-framework/avocado/pull/2626 Can you please help reviewing that PR? And more importantly do you think that with that PR this change is necessary?

Reply:
@ldoktor thanks for looking in to this .will check #2626 and will help to review this as well